apps/selvbetjening-backend/src/main/kotlin/jobs/PubliserInntektsjusteringJobb.kt

```diff
@@ -62,13 +63,24 @@ class PubliserInntektsjusteringJobb(
                 inntektsjusteringService.hentInntektsjusteringForStatus(
                     InntektsjusteringStatus.LAGRET,
                 )
-            nyeInntektsjusteringer.forEach { publiser(it) }
+
+            // Kun midlertidig for å verifisere at fungerer et par ganger i  prod
+            if (featureToggleService.isEnabled(
```

Comment:
Litt vanskelig å forstå hva denne endringen innebærer

Reply:
Ja ser den. Vi har sånn 30+ inntektsendringer liggende i selvbetjening nå fordi vi har skrudd av den tidsstyrte jobben som sender dem over. Når vi skrur på igjen så tenker jeg det er lurt å starte med en av gangen i en runde eller to slik at vi er tryggde på at det fungerer. Etter det kan vi fjerne denne koden helt.